v4/source/system.yaml

```diff
@@ -106,3 +106,20 @@
           $ref: '#/responses/BadRequest'
         '403':
           $ref: '#/responses/Forbidden'
+
+  '/caches/invalidate':
+    post:
+      tags:
+        - system
+      summary: Invalidate all the caches
+      description: |
+        Purge all the cache.
```

Comment:
Can we update this to 'Purge all the in-memory caches for the Mattermost server. This can have  a temporary negative effect on performance while the caches are re-populated.' ?